src/app/template-library/template-details/template-details.component.spec.ts

```diff
@@ -78,11 +78,6 @@ describe('TemplateDetailsComponent', () => {
     expect(fixture.debugElement.query(By.css('.template-title')).nativeElement.innerText).toEqual(template.name);
   });
 
-  it('should call import experience', () => {
-    component.importTemplate('abc123');
-    expect(templateLibraryServiceSpy.importExperience).toHaveBeenCalledWith('abc123');
-  });
-
```

Comment:
Fyi @shawnm0705 - I removed this test because the redirect in the function was breaking the test suite